test/ClangImporter/ctypes_parse.swift

```diff
@@ -31,7 +31,10 @@ func testTribool() {
 func testAnonEnum() {
   var a = AnonConst1
   a = AnonConst2
-#if arch(i386) || arch(arm)
+#if os(Windows)
+  // Windows uses signed enumerations by default
+  _ = a as Int
```

Comment:
This seems concerning; the constants are too big to fit in an Int32, so I'd expect Windows to import this as CLongLong if it's not going to use `unsigned` anything.

Reply:
`cannot convert value of type 'Int' to type 'CLongLong' (aka 'Int64') in coercion` ... oh fun.

Reply:
Okay, that is another Windows-ism.  It defaults to a 32-bit signed value, and you need to explicitly make it 64-bit.  I'll adjust the header for that.